03_array.ipynb

```diff
@@ -133,8 +133,8 @@
    "source": [
     "# Compute sum of large array, one million numbers at a time\n",
     "sums = []\n",
-    "for i in range(0, 1000000000, 1000000):\n",
-    "    chunk = dset[i: i + 1000000]  # pull out numpy array\n",
+    "for i in range(0, 1_000_000_000, 1_000_000):\n",
+    "    chunk = dset[i: i + 1_000_000]  # pull out numpy array\n",
```

Comment:
Adding separators for better readability.

Reply:
Remind me: this syntax came in with py36 or 37?

Reply:
It was introduced in 3.6 as part of [PEP 515](https://www.python.org/dev/peps/pep-0515/).

Either way, I think it's safe to use given that the tutorial environment uses 3.8.

https://github.com/dask/dask-tutorial/blob/121947ac5549f714df8980f756ff2848a105c9d8/binder/environment.yml#L5